fetch.bs

```diff
@@ -4990,20 +4801,20 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
          <li><p>If <var>bytes</var> is failure, then <a lt=terminated for=fetch>terminate</a> the
          ongoing fetch.
 
-         <li><p><a for=ReadableStream>Enqueue bytes</a> given <var>bytes</var> and
-         <var>stream</var>.
+         <li><p><a for=ReadableStream>Enqueue</a> a {{Uint8Array}} wrapping an {{ArrayBuffer}}
```

Comment:
Uint8Array object*
ArrayBuffer object*

And the same below. It's a little unfortunate this duplicates the creation of these objects again, but that's probably fine for now. At some point we probably want an operation that you give a byte sequence and a realm and it returns this.

Reply:
I don't think adding the redundant "object" is a good idea. Ultimately it's your spec so the editorial conventions are up to you, but I wouldn't want to see that carry over to HTML, for example.

I agree that eventually we want something better here in Web IDL.

Reply:
It doesn't seem entirely redundant to be specific about instance vs class.